utilities/navx.py

```diff
@@ -1,43 +1,41 @@
 import math
 
-import wpilib
 from robotpy_ext.common_drivers.navx.ahrs import AHRS
-from robotpy_ext.common_drivers.navx.registerio_spi import RegisterIO_SPI
+from wpilib.interfaces import PIDSource
 
-from utilities.functions import constrain_angle
 
+class NavX:
+    """Wrapper of robotpy navx to be inline with BNO055 api.
+    """
 
-class NavX(AHRS):
+    PIDSourceType = PIDSource.PIDSourceType
+
+    def __init__(self):
+        self.ahrs = AHRS.create_spi(update_rate_hz=100)
+        self.pidsource = self.PIDSourceType.kDisplacement
 
     def getAngle(self):
-        raw = super().getAngle()
-        return constrain_angle(-math.radians(raw))
+        """Get navx angle
```

Comment:
D205 1 blank line required between summary line and description
D400 First line should end with a period